docs/source/deep-learning/pytorch/quickstart/pytorch_quickstart.ipynb

```diff
@@ -0,0 +1,1380 @@
+    "## Connect to MLflow Tracking Server\n",
+    "\n",
+    "Before implementing the training loop, we need to configure the MLflow tracking server because we will log data into MLflow during training.\n",
+    "\n",
+    "In this guide, we will use [Databricks Community Edition](https://www.databricks.com/try-databricks#account) as MLflow tracking server. For other options such as using your local MLflow server, please read the [Tracking Server Overview](https://mlflow.org/docs/latest/getting-started/tracking-server-overview/index.html).\n",
+    "\n",
+    "If you have not, please register an account of [Databricks community edition](https://www.databricks.com/try-databricks#account). It should take no longer than 1 min to register. Databricks CE (community edition) is a free platform for users to try out Databricks features. For this guide, we need the ML experiment dashboard for us to track our training progress.\n",
+    "\n",
+    "After successfully registering an account on Databricks CE, let's connnect MLflow to Databricks CE. You will need to enter following information:\n",
+    "\n",
+    "- Databricks Host: https://community.cloud.databricks.com/\n",
+    "- Username: your signed up email\n",
+    "- Password: your password"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "id": "6fa35hgk9XE3"
+   },
+   "outputs": [],
+   "source": [
+    "mlflow.login()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "TKV0SG-Q9ahb"
+   },
+   "source": [
+    "Now you have successfully connected to MLflow tracking server on Databricks CE, and let's give our experiment a name."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "vRPWOj1o9h-V",
+    "outputId": "2cc120fe-5d06-45f0-a83b-985af5d89f8e"
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<Experiment: artifact_location='dbfs:/databricks/mlflow-tracking/1078557169589361', creation_time=1703121702068, experiment_id='1078557169589361', last_update_time=1703194525608, lifecycle_stage='active', name='/mlflow-pytorch-quickstart', tags={'mlflow.experiment.sourceName': '/mlflow-pytorch-quickstart',\n",
+       " 'mlflow.experimentType': 'MLFLOW_EXPERIMENT',\n",
+       " 'mlflow.ownerEmail': 'qianchen94era@gmail.com',\n",
+       " 'mlflow.ownerId': '3209978630771139'}>"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "mlflow.set_experiment(\"/mlflow-pytorch-quickstart\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "TkyEv1qa6Uxe"
+   },
+   "source": [
+    "## Implement the Training Loop\n",
+    "\n",
+    "Now let's define the training loop, which basically iterating through the dataset and applying a forward and backward pass on each data batch."
```

Comment:
Which iterates through the dataset and applies a forward and backward pass on each data batch